src/test/java/org/broadinstitute/hellbender/engine/SequenceDictionaryValidationIntegrationTest.java

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Created by edwardk on 7/6/15.
+ */
+public class SequenceDictionaryValidationIntegrationTest extends CommandLineProgramTest {
+
+    public static final String SeqDictVal_Test_Directory = "src/test/resources/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils/";
+
+    @DataProvider(name = "SeqDictTest")
+    public Object[][] generateSequenceDictionaryTestData() {
+        return new Object[][]{
+                {},
+                {}
+        };
+    }
+
+    @Override
+    public String getTestedClassName() {
+        return "ExampleIntervalWalker";
+    }
+
+    @DataProvider(name="filenames")
+    public Object[][] filenames() {
+        return new String[][]{
+                {"count_variants.vcf"},
+        };
+    }
+
+    @Test
+    public void testSeqDictValCompatible() throws IOException {
+        IntegrationTestSpec testSpec = new IntegrationTestSpec(
+                " -R " + SeqDictVal_Test_Directory + "test.fasta" +
+                        " -I " + SeqDictVal_Test_Directory + "test.sam" +
+                        " -V " + "TestFeatures:" + SeqDictVal_Test_Directory + "test.vcf" +
+                        " -L " + SeqDictVal_Test_Directory + "test.intervals",
+                0,
+                UserException.BadInput.class);
```

Comment:
Why is this test case expecting `UserException.BadInput`? Isn't it testing the case of compatible dictionaries (which should not produce an exception)?